core-tests/src/test/java/org/mule/runtime/core/internal/construct/DefaultFlowBuilderTestCase.java

```diff
@@ -71,7 +74,7 @@ public void buildsSimpleFlow() throws Exception {
     assertThat(flow.getMuleContext(), is(muleContext));
     assertThat(flow.getProcessors(), is(empty()));
     assertThat(flow.getSource(), is(nullValue()));
-    assertThat(flow.getExceptionListener(), sameInstance(muleContext.getDefaultErrorHandler()));
+    assertThat(flow.getExceptionListener(), not(sameInstance(muleContext.getDefaultErrorHandler(Optional.empty()))));
```

Comment:
Why has this changed? Even if `getDefaultErrorHandler` has changed internally, this still needs to test that Flow uses default error handler when one isn't set/